### PR TITLE
Private/soroush mosaddad99/fix deliverable3 demos

### PR DESCRIPTION
**Pull request by soroushMosaddad99**
- dummy data reverted

- small changes made to made the style consistent

- slight changes made to the headers for better aesthetics

### CODE REVIEW

**Comment by soroushMosaddad99** [comment]
I need to mention that the profile icon is not working at the moment as it's not linked to the home page. I made an attempt to address the issue but the colour of the button keeps changing.

**Comment by WilliamChan4** [comment]
Sure. I'll take a look tomorrow morning. 🙂

File: src/frontend/src/pages/home/home.tsx [hunk 1]
```diff
@@ -42,14 +42,31 @@ export const Home: React.FC = () => {
                 <Grid item xs={4}>
                     <Paper elevation={3}>
                         <List component="nav">
-                            <Link
-                                to="/view-challenge"
-                                style={{ textDecoration: "none" }}
-                            >
+                            <Link to="/view-challenge"style={{ textDecoration: "none" }}>
```
Comment (WilliamChan4) [inline src/frontend/src/pages/home/home.tsx:45]:
```suggestion
                            <Link to="/view-challenge" style={{ textDecoration: "none" }}>
```
There should be a space here.

File: src/frontend/src/pages/view-challenge/view-challenge.tsx [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
-
-import AccountCircle from "@material-ui/icons/AccountCircle";
-
 import SubmitAnswerPopup from "./components/submit-answer-popup";
-import NavigationBar from "./components/navigation-bar";
 
+//compiled with warning, are defined but never used
+//import { Link } from "react-router-dom";
+//import AccountCircle from "@material-ui/icons/AccountCircle";
+//import NavigationBar from "./components/navigation-bar";
```
Comment (WilliamChan4) [inline src/frontend/src/pages/view-challenge/view-challenge.tsx:8]: If they aren't used, you should probably get rid of these lines. 
```suggestion
```

File: src/frontend/src/pages/home/home.tsx [hunk 3]
```diff
@@ -42,14 +42,31 @@ export const Home: React.FC = () => {
...
+                                <Link to="/view-challenge" style={{ textDecoration: 'none' }}>
+                                    <ListItem button divider>
+                                        <ListItemText primary="@Lauren" />
+                                    </ListItem>
+                                </Link>
+                                <Link to="/view-challenge" style={{ textDecoration: 'none' }}>
+                                    <ListItem button>
+                                        <ListItemText primary="@Tina" />
+                                    </ListItem>
+                                </Link>
```
Comment (WilliamChan4) [inline src/frontend/src/pages/home/home.tsx:69]: There's some extra whitespace here. 
```suggestion
                            <Link to="/view-challenge" style={{ textDecoration: 'none' }}>
                                <ListItem button divider>
                                    <ListItemText primary="@Ryan" />
                                </ListItem>
                            </Link>
                            <Link to="/view-challenge" style={{ textDecoration: 'none' }}>
                                <ListItem button divider>
                                    <ListItemText primary="@Amir" />
                                </ListItem>
                            </Link>
                            <Link to="/view-challenge" style={{ textDecoration: 'none' }}>
                                <ListItem button divider>
                                    <ListItemText primary="@Lauren" />
                                </ListItem>
                            </Link>
                            <Link to="/view-challenge" style={{ textDecoration: 'none' }}>
                                <ListItem button>
                                    <ListItemText primary="@Tina" />
                                </ListItem>
                            </Link>
```

File: src/frontend/src/pages/view-challenge/view-challenge.tsx [hunk 4]
```diff
@@ -95,13 +85,14 @@ export const ViewChallenge: React.FC = () => {
...
-                    <Typography variant="h4">Sentence:</Typography>
+                    <Typography variant="h6">Sentence</Typography>
                 </Grid>
                 <Grid item>
                     <Paper elevation={1}>
                         <Box p={3}>
                             <Typography>
-                                Word Scramble is a very felicific game!
+                                User submitted sentence that is really long and
+                                will fit to the screen
```
Comment (WilliamChan4) [inline src/frontend/src/pages/view-challenge/view-challenge.tsx:95]: Put this on one line. 
```suggestion
                                User submitted sentence that is really long and will fit to the screen
```

File: src/frontend/src/pages/shared-components/appbar.tsx [hunk 5]
```diff
@@ -28,13 +20,11 @@ export const ApplicationBar: React.FC = () => {
...
                             <Grid item>
-                                <Typography display="inline">
-                                    Logged in As Tina
-                                </Typography>
+                                <Typography display="inline">Username</Typography>
                             </Grid>
                             <Grid item>
                                 <Box p={3}>
-                                    <Typography variant="h6" display="inline">
+                                    <Typography variant="subtitle1" display="inline">
```
Comment (WilliamChan4) [inline src/frontend/src/pages/shared-components/appbar.tsx:27]: The other inline element "Username" doesn't have a variant so this shouldn't either. 
```suggestion
                                    <Typography display="inline">
```

**Comment by soroushMosaddad99** [comment]
thanks William, I've committed your changes and tested them.

**Pull request merged**